sql/core/src/test/scala/org/apache/spark/sql/execution/columnar/compression/IntegralDeltaSuite.scala

```diff
@@ -173,9 +173,7 @@ class IntegralDeltaSuite extends SparkFunSuite {
     }
 
     test(s"$scheme: long random series") {
-      // Have to workaround with `Any` since no `ClassTag[I#JvmType]` available here.
-      val input = Array.fill[Any](10000)(makeRandomValue(columnType))
-      skeleton(input.map(_.asInstanceOf[I#InternalType]))
+      skeleton(Seq.fill[I#InternalType](10000)(makeRandomValue(columnType)))
```

Comment:
This arguably was more a part of my last "Array -> Seq" change; I missed it, but it concerns using correct types. Here the problems arose because this tried to use a type without a ClassTag and that causes a few problems, among them dealing with Arrays. A few workarounds here like using Seq made it cleaner.